patroni/ha.py

```diff
@@ -122,6 +122,14 @@ def bootstrap(self):
 
     def recover(self):
         self.recovering = True
+        if self.has_lock():
+            if self.patroni.config['master_start_timeout'] == 0:
```

Comment:
We certainly should. Somehow forgot to add it. The lack of this update is what got me started on this patch in the first place. :)